src/pages/ni/auxiliaries/Dashboard.vue

```diff
@@ -165,16 +165,29 @@ export default {
       const sector = this.filters.find(s => s._id === sectorId);
       return sector.label;
     },
-    async getcustomersAndDurationByAuxiliary () {
+    async getPaidInterventionStats () {
       const sectors = [];
+      const paidInterventionStats = {}
       for (const sector of this.filteredSectors) {
-        if (this.customersAndDurationByAuxiliary[sector] || !this.auxiliariesDetailsIsOpened[sector]) continue;
+        if (this.paidInterventionStats[sector] || !this.auxiliariesDetailsIsOpened[sector]) continue;
         sectors.push(sector);
+        paidInterventionStats[sector] = [];
       }
       if (!sectors.length) return;
-      const customersAndDurationByAuxiliaryArray = await this.$stats.getCustomersAndDurationByAuxiliary({ sector: sectors, month: this.selectedMonth });
-      for (const customersAndDurationByAuxiliary of customersAndDurationByAuxiliaryArray) {
-        this.$set(this.customersAndDurationByAuxiliary, customersAndDurationByAuxiliary.sector, customersAndDurationByAuxiliary.customersAndDuration);
+      const paidInterventionStatsByAuxiliary = await this.$stats.getPaidInterventionStats({ sector: sectors, month: this.selectedMonth });
+      for (const paidInterventionStatsForOneSector of paidInterventionStatsByAuxiliary) {
+        const auxiliarySectors = paidInterventionStatsForOneSector.sectors.reduce((acc, auxiliarySectorsArray) => {
```

Comment:
tu peux faire une boucle for directement non ?
les `reduce` ne sont pas tres lisible ni opti